crates/fluvio-cli/src/error.rs

```diff
@@ -112,6 +112,12 @@ impl CliError {
                 }
                 _ => Err(self),
             },
+            Self::ClientError(FluvioError::Socket(SocketError::Io(io)))
+                if io.kind() == ErrorKind::TimedOut =>
+            {
+                println!("Timed out while waiting on socket response");
```

Comment:
This is bit technical.  Should be something like: "Network connection timed out while waiting response"